src/Components/Publishing/Layouts/ArticleWithFullScreen.tsx

```diff
@@ -90,3 +50,20 @@ export class ArticleWithFullScreen extends React.Component<
 }
 
 export default track()(ArticleWithFullScreen)
+
+export const getSlideshowImagesFromArticle = (article: ArticleData) => {
+  const fullscreenImages = []
+  let sectionIndex = 0
+
+  map(article.sections, section => {
```

Comment:
Are we iterating over an object, or an array? If an array, we should be be using native `article.sections.map(...)` methods rather than an external lib. If over an object, then `map`, from lodash, is useful.